android/src/main/java/io/customer/reactnative/sdk/logging/CustomerIOReactNativeLogger.kt

```diff
@@ -0,0 +1,91 @@
+package io.customer.reactnative.sdk.logging
+import com.facebook.react.bridge.Arguments
+import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.bridge.ReactContextBaseJavaModule
+import com.facebook.react.bridge.ReactMethod
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.module.annotations.ReactModule
+import com.facebook.react.modules.core.DeviceEventManagerModule
+import io.customer.sdk.core.util.CioLogLevel
+import kotlin.collections.buildMap
+import io.customer.sdk.core.di.SDKComponent
+import io.customer.sdk.core.util.Logger
+
+@ReactModule(name = CustomerIOReactNativeLoggingEmitter.NAME)
+class CustomerIOReactNativeLoggingEmitter(
+    private val reactContext: ReactApplicationContext,
+) : ReactContextBaseJavaModule(reactContext) {
+    companion object {
+        const val EVENT_NAME = "CioLogEvent"
+        const val NAME = "CioLoggingEmitter"
+
+    }
+    private var listenerCount = 0
+
+    override fun getName(): String {
+        return NAME
+    }
+
+    @ReactMethod
+    fun addListener(eventName: String) {
+        listenerCount++
+    }
+
+    @ReactMethod
+    fun removeListeners(count: Int) {
+        listenerCount -= count
+    }
+
+    fun sendEvent(params: WritableMap) {
+        reactContext
+            .getJSModule(DeviceEventManagerModule.RCTDeviceEventEmitter::class.java)
+            .emit(EVENT_NAME, params)
+    }
+}
+
+class CustomerIOReactNativeLoggingWrapper private constructor(
+    private var moduleRegistry: ReactApplicationContext,
+    override var logLevel: CioLogLevel): Logger {
+
+    private val emitter: CustomerIOReactNativeLoggingEmitter?
+        get() = moduleRegistry?.getNativeModule(CustomerIOReactNativeLoggingEmitter::class.java)
+
+    companion object {
+        fun getInstance(moduleRegistry: ReactApplicationContext, logLevel: CioLogLevel): CustomerIOReactNativeLoggingWrapper {
+            val instance = CustomerIOReactNativeLoggingWrapper(moduleRegistry, logLevel)
+            SDKComponent.overrideDependency<Logger>(instance)
+            return instance
+        }
+    }
+
+    override fun debug(message: String) {
+        emit(message, CioLogLevel.DEBUG)
+    }
```

Comment:
```suggestion
        override fun debug(message: String) = emit(message, CioLogLevel.DEBUG)
```

can be simplified here and other places